test/crio-wipe.bats

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env bats
+
+# this test suite tests crio wipe running with combinations of cri-o and
+# podman.
+
+load helpers
+PODMAN_BINARY=${PODMAN_BINARY:-$(command -v podman || true)}
+
+function setup() {
+	setup_test
+	export CONTAINER_VERSION_FILE="$TESTDIR"/version.tmp
+	export CONTAINER_VERSION_FILE_PERSIST="$TESTDIR"/version-persist.tmp
+}
+
+function run_podman_with_args() {
+	if [[ ! -z "$PODMAN_BINARY" ]]; then
+		run "$PODMAN_BINARY" --root "$TESTDIR/crio" --runroot "$TESTDIR/crio-run" "$@"
+		echo "$output"
+		[ "$status" -eq 0 ]
+	fi
+}
+
+function teardown() {
+	cleanup_test
+	run_podman_with_args stop -a
+	run_podman_with_args rm -fa
+}
+
+# run crio_wipe calls crio_wipe and tests it succeeded
+function run_crio_wipe() {
+	run $CRIO_BINARY --config "$CRIO_CONFIG" wipe
```

Comment:
```suggestion
	run $CRIO_BINARY_PATH --config "$CRIO_CONFIG" wipe
```

This :point_up: should fix the static binary CI

Reply:
AH you're the best thanks